eng/pipelines/templates/jobs/ci.yml

```diff
@@ -314,23 +315,19 @@ jobs:
       - pwsh: |
           $failed = $false
           if ("$(ProjectNames)") {
-            $services = "$(ChangedServices)" -split ","
-
-            foreach ($service in $services) {
-              Write-Host "building $service"
-              dotnet build eng/service.proj -warnaserror `
+            dotnet build eng/service.proj -warnaserror `
```

Comment:
Calling out that this is incorrect. I need to filter the projectlistoverridefile to just the projects where `CheckSnippets` is enabled.